config/spring19_seminar/cola_var/conf_gen.py

```diff
@@ -0,0 +1,78 @@
+import scratch
```

Comment:
What is this doing that we can't do with overrides and one or two config files? I worry that this'll just make debugging and comparison harder.

Reply:
It generates the two config files in question and command lines with desired overrides

Reply:
I'd suggest keeping this private and just checking in the config files and bash files. They'll be much easier to read and much more standard for jiant.

Reply:
definitely:) this was for the original research questions where I thought there might be much more pretrain/model configurations to compare

Reply:
Yeah. Mind removing for now? This is pretty close to being ready to merge in.

Reply:
Done!